clients/src/main/java/org/apache/kafka/common/Cluster.java

```diff
@@ -45,14 +46,16 @@
      */
     public Cluster(Collection<Node> nodes,
                    Collection<PartitionInfo> partitions,
-                   Set<String> unauthorizedTopics) {
-        this(false, nodes, partitions, unauthorizedTopics);
+                   Set<String> unauthorizedTopics,
+                   Set<String> internalTopics) {
```

Comment:
Yes, looks that way sadly. I was planning to check that and also if we need more tests before asking for a review, but you were too fast (I am certainly not complaining though :)).